peps/pep-0007.rst

```diff
@@ -170,6 +184,9 @@ Naming conventions
 * Macros should have a MixedCase prefix and then use upper case, for
   example: ``PyString_AS_STRING``, ``Py_PRINT_RAW``.
 
+* Macro parameters should use ``ALL_CAPS`` style,
+  so they are easily distinguishable from C variables and struct members.
```

Comment:
You can explain that they are not regular variables, preprocessor arguments are more just plain textw they usually require to be surrounded by parenthesis, and so should be easy to spot.

Reply:
I'm not sure we should introduce such explanations in PEP-7. Currently, it is a style guide only. I'm happy to add a sentence or two about it if everyone else thinks it is a good idea, though :)

Reply:
Encouraging the redundant parentheses would be in scope, though.

Reply:
> Encouraging the redundant parentheses would be in scope, though.

I'd prefer if we could address that in a follow-up PR. Marking this as resolved.